test/test_ops.py

```diff
@@ -693,6 +738,30 @@ def get_fn_args(self, device, contiguous, batch_sz, dtype):
 
         return x, weight, offset, mask, bias, stride, pad, dilation
 
+    def make_obj(
+        self, device, contiguous, batch_sz, dtype, in_channels=6, out_channels=2, kernel_size=(3, 2), groups=2
+    ):
+        _, _, _, _, _, stride, padding, dilation = self.get_fn_args(device, contiguous, batch_sz, dtype)
```

Comment:
The `get_fn_args()` can be quite slow. Since you only care about the stride, padding and dilation which are hardcoded in the method, you can just set their values.